src/main/java/org/jenkinsci/test/acceptance/po/WizardCustomizeJenkins.java

```diff
@@ -0,0 +1,74 @@
+package org.jenkinsci.test.acceptance.po;
+
+import static org.jenkinsci.test.acceptance.Matchers.hasContent;
+
+import java.util.concurrent.Callable;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.NoSuchFrameException;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Page object for Wizard Customize Jenkins Page.
+ *
+ */
+public class WizardCustomizeJenkins extends PageObject {
+
+    public WizardCustomizeJenkins(Jenkins jenkins) {
+        super(jenkins.injector, jenkins.url(""));
+    }
+
+    public WizardCustomizeJenkins doInstallSuggested() {
+        By suggestedButtonLocator = By.partialLinkText("Install suggested plugins");
```

Comment:
I can do that, but in this case I'm not creating a new selector, just using an existing one... or maybe I'm not correctly understanding you suggestion.